docs/device-emulation.md

```diff
@@ -69,3 +69,33 @@ For example:
     Decompressing…           [***************************************]
     Waiting…                 [***************************************]
     Hughski ColorHug2: OK!
+
+## Pcap file conversion
+
+Emulation can also be used during the development phase of the plugin if the hardware is not
+available or to reduce the testing time. But this requires a way to create the emulation file while
```

Comment:
probably also helps with the fact you don't need to do a write cycle with the device while writing the code too.   Nice!